src/pages/gd/Portfolio/index.tsx

```diff
@@ -187,12 +189,11 @@ const Portfolio = () => {
             </Card>
             <PortfolioTitleSC className="pl-2 mb-3">{i18n._(`Positions`)}</PortfolioTitleSC>
             <Card contentWrapped={false} style={{position:'relative'}}>
-            {data?.list.map(stake => (
-               stake.protocol === LIQUIDITY_PROTOCOL.GOODDAO && !stake.isV2 && (
-                  <AppNotice text={newGovString} 
+            {showNotice?.indexOf(true) !== -1 && (
```

Comment:
probably should be a boolean

Reply:
it is, see comment above

Reply:
I actually could add this indexOf to the end of the map
so the boolean is checked in the declaration of showNotice